initial commit: проверить взаимодействие контейнеров

NekhoroshkinaElena opened the pull request.

NekhoroshkinaElena committed:
initial commit: проверить взаимодействие контейнеров

NekhoroshkinaElena committed:
initial commit: проверить взаимодействие контейнеров

NekhoroshkinaElena committed:
feat: перенести валидацию данных в Gateway

johnskyi commented on server/pom.xml:
```diff
@@ -0,0 +1,132 @@
+		</dependency>
+	</dependencies>
+
+<!--	<dependencies>-->
```
если этот код не нужен, удалите пожалуйста

johnskyi commented on gateway/src/main/java/ru/practicum/shareit/booking/BookingController.java:
```diff
@@ -0,0 +1,72 @@
+@RequiredArgsConstructor
+@Slf4j
+@Validated
+
```
лишнюю пустую строку лучше убрать

johnskyi commented on gateway/src/main/java/ru/practicum/shareit/exception/ErrorHandler.java:
```diff
@@ -0,0 +1,21 @@
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestControllerAdvice;
+
+@RestControllerAdvice
```
было бы неплохо добавить здесь логирование и выводить ошибки в log.debug или log.error

NekhoroshkinaElena committed:
fix: удалить лишнюю строку в классе BookingController
feat: дабвить логирование в класс ErrorHandler
fix: удалить лишний код в pom.xml (server)

NekhoroshkinaElena commented:
Добрый вечер, исправила ошибки:)
